about.html

```diff
@@ -0,0 +1,158 @@
+            <a href="#program" class="nav-link">program</a>
+          </li>
+          <li class="nav-item">
+            <a href="#speakers" class="nav-link">speakers</a>
+          </li>
+        </ul>
+        <a href="">
+          <button class="nav-btn btn">CC Compaign</button>
+        </a>
+      </nav>
+      <div class="hero-body">
+        <img
+          class="about-bg-img"
+          src="./images/about_title.png"
+          alt="about hero background"
+        />
+        <div class="about-hero-underline"></div>
+        <p class="hero-subtitle">
+          The Creative Commons Global Summit , held every two years, is a place
+          where CC activists from all over the world and experts, scholars and
+          activists from all walks of life who support and practice open culture
+          gather together.
+        </p>
+        <p class="contact">
+          Please contact us via email for any further question about CC Global
+          Submit 2015! <br /><a href="#">ccsummit@gmail.com </a>
+        </p>
+      </div>
+    </section>
+    <hr />
+    <section class="about-section">
+      <h2 class="about-title">Creative Commons Global Submit 2015 logo</h2>
+      <div class="about-underline"></div>
+      <div class="paragraph-container">
+        <p class="about-subtitle">
+          The winner of the logo contest, which was held from June 8th to July
+          7th, was decided through online voting and evaluation by the judges.
+          After consultation with the winning designer, the logo of CC Global
+          Summit 2015, which was optimized for use on the website and
+          promotional materials, is as follows.
+        </p>
+        <img class="ccsubmit-logo" src="./images/about-logo.png" alt="" />
+      </div>
+    </section>
+    <hr />
+    <section class="previous-submit">
+      <h2 class="ps-title">See the past CC Global Submit</h2>
+      <div class="about-underline"></div>
+      <p class="ps-subtitle">Take a look at the last two CC Global Submit.</p>
+      <div class="previous-images">
+        <img src="./images/about_2011.png" alt="robs image" />
+        <img src="./images/about_2013.png" alt="bob image" />
+      </div>
+    </section>
+    <!-- footer -->
+    <footer>
+      <div id="Sponsors" class="footer-partners">
+        <header class="footer-header">
+          <h3 class="footer-header-title">Partners</h3>
+          <div class="footer-header-underline underline"></div>
+        </header>
+        <div class="partner-container">
+          <a href="#" class="logo-partner">
+            <img src="images/footer-img.JPG" alt="mozilla images" />
+          </a>
+        </div>
+      </div>
+      <div class="copywrite">
+        <a href="#">
+          <img class="footer-logo" src="images/logo.png" alt="footer logo" />
+        </a>
+        <p class="copywrite-text">
+          2015 Creative Commons Korea. Some Rights Reserved.
+        </p>
+      </div>
```

Comment:
- [ ] **[OPTIONAL]** In the original design the footer is dark. Please 🙏🏽 correct it to look like the original design.
![image](https://user-images.githubusercontent.com/58642949/190402630-73e19d21-9a2e-45e2-8112-657a3e96df66.png)